### DISCUSSION TITLE
How to get a refund

### DISCUSSION THREAD

**Discussion by kics223w1** [cssbattle/forum#388] [Q&A]
Hi CSSBattle Dev,

I've been busy lately so I don't have enough time for CSSBattle. That is why I canceled my subscription yesterday.  But I'm still a PLUS member and the membership will expire on Sat Apr 13, 2024.

Is there any way to get the refund and remove my PLUS membership immediately 🤔 .  If yes, please give me the email so I can send you my information payment.

Thank you and have a good day ❤️

**Comment by chinchang** [comment]
Hi @kics223w1 Please send us a mail on contact@cssbattle.dev

**Reply by kics223w1** [reply]
HI @chinchang, I've sent you a mail 👍🏻
